lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -372,8 +372,9 @@ def max_thread_limit
   begin
     require 'parallel_tests'
 
-    args = ['-t', 'rspec'].
-      concat(Rake::FileList[pattern].to_a)
+    args = ['-t', 'rspec']
+    args = args.push('--').concat(ENV['CI_SPEC_OPTIONS'].split 'j').push('--') unless ENV['CI_SPEC_OPTIONS'].nil?
```

Comment:
These next two lines don't need the `=` assignment, push and concat should modify the object directly.